packages/back-end/types/event-webhook-log.d.ts

```diff
@@ -0,0 +1,10 @@
+export interface EventWebHookLogInterface {
+  id: string;
+  eventWebHookId: string;
+  organizationId: string;
+  dateCreated: Date;
+  responseCode: string | null;
+  error: string | null;
```

Comment:
I think this needs to change to `responseBody` instead of `error` to match the mongoose model.

Reply:
I've fixed the fields for the EventWebHookLog model and interface here 68e6d5f1